src/include/printer/autoconfig.rb

```diff
@@ -108,9 +107,7 @@ def initAutoconfig(key)
             Printerlib.client_conf_server_name
           )
         else
-          if !Printerlib.ExecuteBashCommand(
-              Ops.add(Printerlib.yast_bin_dir, "cups_client_only none")
-            )
+          if !Printerlib.ExecuteBashCommand(Printerlib.yast_bin_dir + "cups_client_only none")
```

Comment:
this is a bit hard to read. Maybe use instead `#{File.join(Printerlib.yast_bin_dir, "cups_client_only")} none` or even better define path at first like
```
script = File.join(Printerlib.yast_bin_dir, "cups_client_only")
if !Printerlib.ExecuteBashCommand("#{script} none")
```

so it is obvious what is script and what is argument.

Reply:
Well, no. Not only no, but **hell, no**.

If the Unix/Linux path separator ever changes, we are completely screwed anyway. But there is no chance in hell that this will ever happen. And that File.join() business just makes code **MUCH** harder to read. It distracts from what's going on here.

I almost wanted to create a dedicated function for this common case to call a binary from yast_bin_dir, but this would also add to the diff size without much gain. I decided against it.